conf/sources.json

```diff
@@ -3,14 +3,16 @@
     "prefix_cluster1_stream_alert_kinesis": {
       "logs": [
         "cloudwatch",
-        "ghe"
+        "ghe",
+        "osquery"
       ]
     }
   },
   "s3": {
     "prefix.cluster.sample.bucket": {
       "logs": [
-        "cloudtrail"
+        "cloudtrail",
```

Comment:
I think cloudtrail would more appropriately go under `kinesis` above. Can you update here and within your test events?